src/util/SE3.py

```diff
@@ -61,6 +63,18 @@ def from_tf_tree(cls, tf_buffer: tf2_ros.Buffer, parent_frame: str, child_frame:
         result = SE3(position=numpify(tf_msg.translation), rotation=SO3(numpify(tf_msg.rotation)))
         return result
 
+    @classmethod
+    def from_se3_time(cls, tf_buffer: tf2_ros.Buffer, parent_frame: str, child_frame: str) -> Tuple[SE3, Time]:
```

Comment:
can you rename this to 'from_tf_tree_time' or something so it fits the other naming conventions